tests/sentry/utils/suspect_resolutions/test_project_issue_correlation.py

```diff
@@ -0,0 +1,80 @@
+import random
+from unittest import mock
+
+from django.utils import timezone
+
+from sentry.models import Commit, CommitFileChange, GroupRelease, GroupStatus, ReleaseCommit
+from sentry.testutils import TestCase
+from sentry.utils.suspect_resolutions.commit_correlation import is_issue_commit_correlated
+from sentry.utils.suspect_resolutions.metric_correlation import is_issue_error_rate_correlated
+from sentry.utils.suspect_resolutions.project_issue_correlation import (
+    get_project_issues_with_correlated_commits_and_error_rate,
+)
+from tests.sentry.utils.suspect_resolutions.test_metric_correlation import (
+    END,
+    START,
+    WINDOW,
+    generate_issue_error_series,
+)
+
+
+class TestProjectIssueCorrelation(TestCase):
+    @mock.patch("sentry.tsdb.get_range")
+    def test_project_issue_correlation(self, mock_get_range):
```

Comment:
Hmm, this test feels like its doing a lot. To me it looks like we're making sure the groups are commit-correlated and metric-correlated, but we have tests that individually test for that in `TestMetricCorrelation` and `TestCommitCorrelation`. 

What do we really want to be testing here?

Reply:
This test was supposed to be a simple happy-path case where the unresolved issue is correlated by both commit data and error rates. I think we want to test that the function will successfully identify suspect resolutions given various scenarios - I'll add additional tests to cover different cases.

Reply:
What if you mocked out `is_issue_error_rate_correlated` and `is_issue_commit_correlated` to return true? So instead of setting everything up in such a way where all the data would return true for both functions, we just mock it. 

That way we isolate what we wanna test to just logic contained in `get_project_issues_with_correlated_commits_and_error_rate`.

My main point is trying to avoid repeated testing of functions which already have coverage. I.e. we already have tests for `is_issue_error_rate_correlated` and `is_issue_commit_correlated`, what do we really want to be testing within `get_project_issues_with_correlated_commits_and_error_rate`?